.claude/hooks/gemini-context-injector.sh

```
@@ -0,0 +1,129 @@
+
+# Ensure log directory exists
+mkdir -p "$(dirname "$LOG_FILE")"
+
+# Read input from stdin
+INPUT_JSON=$(cat)
+
+# Function to log injection events
+log_injection_event() {
+    local event_type="$1"
+    local details="$2"
+    local timestamp=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)